site/static/examples/ingress/kong/deployment.patch.json

```diff
@@ -0,0 +1,37 @@
+{
+  "spec": {
+    "template": {
+      "spec": {
+        "containers": [
+          {
+            "name": "proxy",
+            "ports": [
+              {
+                "containerPort": 8000,
+                "hostPort": 80,
+                "name": "proxy",
+                "protocol": "TCP"
+              },
+              {
+                "containerPort": 8443,
+                "hostPort": 43,
+                "name": "proxy-ssl",
+                "protocol": "TCP"
+              }
+            ]
+          }
+        ],
+        "nodeSelector": {
+          "ingress-ready": "true"
+        },
+        "tolerations": [
+          {
+            "effect": "NoSchedule",
+            "key": "node-role.kubernetes.io/master",
```

Comment:
@neolit123 advice here please

Reply:
this should be either:
https://github.com/kubernetes-sigs/kind/blob/main/site/static/examples/ingress/contour/patch.json#L8-L18

or:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/addons/proxy/manifests.go#L111-L112

the former tolerates only the CP node taints for kubeadm 1.24, the latter tolerate all taints (i.e. a wildcard).